src/targets/gpu/mlir.cpp

```diff
@@ -580,12 +599,93 @@ struct mlir_program
 
     std::string get_tune_params(bool xdlops) { return get_mlir_perf_for_conv(pp, xdlops); }
 
+    void tuning_table_create()
+    {
+        tuning_table               = mlirRockTuningTableCreate();
```

Comment:
Isnt there a `mlirRockTuningTableDestroy` that needs to be called?

Reply:
Yes there is .. and it should be called -- now done.